platform/mellanox/fw.mk

```diff
@@ -34,6 +34,8 @@ else
 SONIC_ONLINE_FILES += $(MLNX_FW_FILES)
 endif
 
+MLNX_FILES += $(MLNX_FW_FILES)
```

Comment:
>MLNX_FILES [](start = 0, length = 10)

export it? many occurrences in this PR.

Reply:
@qiluo-msft We need to keep as is because there is a logic for separate copying of these files:
https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/sonic_debian_extension.j2#L465

The motivation for these changes is to introduce a single variable for ONE image dependencies:
https://github.com/Azure/sonic-buildimage/blob/master/platform/mellanox/one-image.mk#L14
This one goes as a Build System variable with aggregator role.